src/main/java/dude/task/ListManager.java

```diff
@@ -31,6 +31,10 @@ public static void printList() {
         System.out.println(Interface.LINE);
     }
 
+    public static Task getString(int index){

```

Comment:
coding standard: there should be space between the brackets
`public static Task getString(int index) {`